package.json

```diff
@@ -25,7 +25,7 @@
     "test:coverage": "nyc --reporter=lcov --reporter=text-summary yarn test",
     "test:browser:report": "node ./test/utils/multiReportGenerator.js",
     "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5040 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json",
-    "test:browser:ci": "npm-run-all -p -r start:ci mocks test:browser && npm run test:browser:report"
+    "test:browser:ci": "IPV_BASE_URL=http://localhost:5030 npm-run-all -p -r start:ci mocks test:browser && npm run test:browser:report"
```

Comment:
What do we need `IPV_BASE_URL` for in BAV?

Reply:
IPV_BASE_URL is used in the goTo() function in [relyingPartyPage.js](https://github.com/govuk-one-login/ipv-cri-bav-front/blob/main/test/browser/pages/relyingPartyPage.js) to start the stub service and navigate to the AuthorizeLocation URL

Reply:
That file looks like it's using `IPV_STUB_URL` rather than `IPV_BASE_URL` unless I'm missing something?

Reply:
No, you're right. We don't need this in our browser tests script, I've removed it